en: refine contents about local-storage (#1053)
pingcap/docs-tidb-operator: en/configure-storage-class.md

## Local PV configuration

Kubernetes currently supports statically allocated local storage. To create a local storage object, use `local-volume-provisioner` in the [local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) repository.

> The following process uses `/mnt/disks` as the discovery directory and `local-storage` as the StorageClass name to create a PV. If you need to use a different data disk or StorageClass for monitoring, backup, etc., refer to the following [example](#disk-mount-examples) for configuration.

Reviewer: ```suggestion
The following process uses `/mnt/disks` as the discovery directory and `local-storage` as the StorageClass name to create a PV. If you need to use a different data disk or StorageClass for monitoring, backup, or other purposes, refer to the following [example](#disk-mount-examples) for configuration.
```
Please avoid using Latin abbreviations.

    kubectl get pv | grep local-storage
    ```

    `local-volume-provisioner` creates a PV for each mounting point under the discovery directory.

    > **Note:**
    >

>     > - on GKE, `local-volume-provisioner` creates a local volume of only 375 GiB in size by default.

Reviewer: ```suggestion
    > - On GKE, `local-volume-provisioner` creates a local volume of only 375 GiB in size by default.
```

>     > - If no mount point is found in the directory, no PV is created and the output of `kubectl get pv | grep local-storage` is empty.

Reviewer: ```suggestion
    > - If no mount point is in the discovery directory, no PV is created and the output of `kubectl get pv | grep local-storage` is empty.
```
